content/TrueCommand/TCGettingStarted/AdminAccounts.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Creating the Admin Account"
+description: "This article describes how to create the administrator account in TrueCommand."
+weight: 30
+tags:
+- tcinstall
+- tcadmin
+- tcresetpwd
+- scaletoptoolbar
+- coretoptoolbar
+---
+{{< toc >}}
+
+When accessing the interface for the first time, you need to create an admin account. 
+
+![TrueCommand Admin Creation](/images/TrueCommand/2.0/FirstLogin.png "TrueCommand Admin Creation")
+
+Enter a username and password. Read the Terms of Service, select **I have read and agree to the terms of service**, and click **SIGN UP**.
+
+TrueCommand creates the admin login credentials and displays the login page.
+
+![TrueCommand Admin Sign In](/images/TrueCommand/2.0/LoginAdmin.png "TrueCommand Admin Sign In")
+
+You can now log in to the TrueCommand web interface with the new administrator account credentials.
+
+### Resetting a User Password on the Login Screen
+
+{{< include file="static/includes/TrueCommand/2.1/ResettingUserPassword.md.part" markdown="true" >}}
```

Comment:
You can add for Cloud admin and other user passwords can be reset through support as well. Admin password resets will generate a temporary password that'll be used once to login, whereupon they'll need to change their password.